bakkesmod-playlist-rank-viewer/RankAndPlaylistData.h

```diff
@@ -0,0 +1,74 @@
+#pragma once
+
+#include "pch.h"
+#include "Enums.h"
+
+struct Color { int r, g, b; };
+struct PlayerStatStruct { Color color; std::string text; };
+struct RankInfo { Color color; std::string name; };
+
+map<PLAYLIST, string> playlistNames = {
+	{UNRANKED, "Casual"},
+	{SOLO, "Solo"},
+	{TWOS, "Doubles"},
+	{THREES, "Standard"},
+	{HOOPS, "Hoops"},
+	{RUMBLE, "Rumble"},
+	{DROPSHOT, "Dropshot"},
+	{SNOWDAY, "Snowday"},
+	{TOURNAMENT, "Tournament"}
+};
+
+array<PLAYLIST, 9> playlistsToCheck = { UNRANKED, SOLO, TWOS, THREES, HOOPS, RUMBLE, DROPSHOT, SNOWDAY, TOURNAMENT };
+
+map<RANK, RankInfo> RankTiers = {
```

Comment:
Should we delete this version?

Reply:
I was considering it, but I didn't want to go too crazy so you can still have some options. Totally up to you. As of right now, it's not being used in the component but does act as a full-length list of the tiers.

Reply:
I reduced these 2 maps into a single map in my last commit. I think this makes a lot more sense since we can just store both versions of the string in the same map

Reply:
I like it